lib/util.js

```diff
@@ -2,8 +2,12 @@
 
 const _ = require('lodash')
 const call = require('call')
+const Controller = require('trails-controller')
+const Policy = require('trails-policy')
 const lib = require('.')
 
+const isClassStandardized = {} // cache to know if class is standar or not
```

Comment:
@tjwebb I remove totally the cache and instead adding a boolean to the class itself.
But in order to this to work we need to create a `trails-standardized-controller` and `trails-standardized-policy`, like this the current one is used for specific webserver api and this ones for unified api. Can I create them ?